FarsiLibrary.Win/Controls/FADatePicker.cs

```diff
@@ -465,8 +465,11 @@ public void ResetSelectedDateTime()
 
         protected override void OnMouseWheel(MouseEventArgs e)
         {
-            if (mv.Visible == false && (this.FormatInfo == FormatInfoTypes.DateShortTime || this.FormatInfo == FormatInfoTypes.ShortDate))
+            if (mv.Visible == false && 
+                (this.FormatInfo == FormatInfoTypes.DateShortTime || this.FormatInfo == FormatInfoTypes.ShortDate) &&
+                mv.MonthViewControl.DefaultCulture.Equals(mv.MonthViewControl.PersianCulture))
```

Comment:
It probably needs separate 'components' for the date selection (one per calendar type) to properly support the scrolling feature for other calendars. For English/Arabic it can be a no-op implementation. 

Maybe this logic can be encapsulated as: 

- If not visible, return
- If visible, ask the component for the current culture. We can have a dictionary where the key is the culture (or culture name), and the value is a service, e.g. `CalendarScroller`. The service can then answer if supports scrolling:
  - For Persian, the related code can be moved to the PersianCalendarScroller class.
  - For Arabic and English, we'll need a No-Op class, that returns false in their "CanScroll" method.
  - The OnMouseWheel method would call the `CanScroll` to see if it can scroll.

Does this make sense? Let me know if you want me to do it.

Reply:
Yes, That's a good idea to separate component for CalendarScroller. I will work on that and make the changes and after that I can implement CalendarScroller for English/Arabic as well.